scripts/upgrade

```diff
@@ -151,45 +154,42 @@ ynh_store_file_checksum --file="$final_path/searx/settings.yml"
 # SECURING FILES AND DIRECTORIES
 #=================================================
 
-chown $app: --recursive "$final_path"
+chown -R $app: "$final_path"
 
 #=================================================
 # CONFIGURE UWSGI FOR SEARX
 #=================================================
 ynh_script_progression --message="Reconfiguring uWSGI for Searx..."
 
-cp ../conf/searx.ini /etc/uwsgi/apps-available/$app.ini
-ynh_replace_string --match_string="__APP__" --replace_string="$app" --target_file=/etc/uwsgi/apps-available/$app.ini
-ynh_replace_string --match_string="__FINALPATH__" --replace_string="$final_path" --target_file=/etc/uwsgi/apps-available/$app.ini
+# Clean old files
+ynh_secure_remove --file="/etc/uwsgi/apps-enabled/$app.ini"
+ynh_secure_remove --file="/etc/uwsgi/apps-available/$app.ini"
+ynh_add_uwsgi_service
 
 #=================================================
-# CHECK SEARX STARTING
+# ADVERTISE SERVICE IN ADMIN PANEL
 #=================================================
-ynh_script_progression --message="Restarting Searx..." --weight=3
 
-# Wait for searx to be fully started
-ynh_systemd_action --service_name=uwsgi --action=restart --line_match="spawned uWSGI master process" --log_path="/var/log/uwsgi/app/$app.log"
+# Ajoute le service au monitoring de YunoHost.
```

Comment:
c.f. previous change, it's really not just about "monitoring" or "advertising the service in the webadmin", this is a general integration that allows monitoring (yes) and the admin to interact with the service and easily share the logs from the webadmin and CLI if necessary

(Sorry I know these are just comments and I'm a bit pedantic on the semantic but I like when we know why we're doing things :s)

```suggestion
```